### PR TITLE
Post/display part of rails template with different locale

### PR DESCRIPTION
**Pull request by Ikera**
@basti @JovicaSusa @codalicious @draganf @marijanam Show part of a Rails template in other language.

### CODE REVIEW

File: source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+categories: 
+  - Ruby and Rails
+tags: 
+  - ruby
+  - rails
+  - I18n.locale
+published: true
+---
+
+I recently worked on a Rails project, which had parts of pages in different languages. That may be a problem if you have already translated their entire text to all required languages. You can even be tempted to hardcode parts of the text into other languages. Fortunately, there is an elegant way to solve that problem, just wrap parts of template or partials into blocks with different locale, like this:
```
Comment (marijanam) [inline source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown]: @Ikera  did you maybe mean 'wrap parts of template or partials into block with desired locale'?

**Comment by basti** [comment]
👍

File: source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+# in config/locales/fr.yml
+fr:
+  my_great_header: "My French great header"
+  first_paragraph: "First French paragraph"
+  second_paragraph: "Second French paragraph"
+```
+
+<!--more-->
+
+And client want first paragraph to be always on English.
```
Comment (marijanam) [inline source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown]: @Ikera and client wants first paragraph to always be in English.

**Comment by marijanam** [comment]
👍

**Comment by JovicaSusa** [comment]
@Ikera 👍

File: source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+Suppose, there is a template with only header and two paragraphs.
+
+``` 
+<h1><%= t('my_great_header') %></h1>
+
+<p><%= t('first_paragraph') %></p>
+
+<p><%= t('second_paragraph') %></p>
+```
+And locale on English and French for that template.
```
Comment (codalicious) [inline source/_posts/2016-05-28-display-part-of-rails-template-with-different-locale.markdown]: @Ikera it should be "in English"

**Comment by codalicious** [comment]
@Ikera 👍

**Pull request merged**